docs/analytics/quickstart.mdx

```diff
@@ -58,18 +58,18 @@ or deletes.
 
 ## Known Limitations
 
-`deltalake` tables are currently in beta. The following is a list of known limitations. Many of these
-will become resolved as `deltalake` tables become production-ready.
+`parquet` tables are currently in beta. The following is a list of known limitations. Many of these
+will become resolved as `parquet` tables become production-ready.
 
 - [ ] `UPDATE` statements
 - [ ] Nested `DELETE` statements
 - [ ] Partitioning tables by column
 - [ ] Some Postgres types like arrays, JSON, time, and timestamp with time zone
 - [ ] User-defined functions, aggregations, or types
-- [ ] Referencing `deltalake` and regular Postgres `heap` tables in the same query
+- [ ] Referencing `parquet` and regular Postgres `heap` tables in the same query
 - [ ] Write-ahead-log (WAL) support and `ROLLBACK`
 - [ ] Foreign keys
 - [ ] Index scans
 - [ ] `TEMP` tables
 - [ ] Using an external data lake as a table storage provider
-- [ ] Full text search over `deltalake` tables with `pg_bm25`
+- [ ] Full text search over `parquet` tables with `pg_bm25`
```

Comment:
Here I would also add a bullet for "query remote parquet files stored in Delta Lake/Iceberg"